packages/core/src/Site/index.ts

```diff
@@ -378,15 +376,22 @@ export class Site {
   }
 
   /**
-   * Converts an existing GitHub wiki or docs folder to a MarkBind website.
+   * Creates the default layout of the website based on a template.
    */
-  async convert() {
+  async generateTemplateDefault(templateConfig: TemplateConfig, isConversion: boolean) {
```

Comment:
Since this method now handles the default file generation for `markbind init` and conversion for `markbind init --convert`, a name like `initSite` might be more suitable (and self-explanatory), as compared to `generateTemplateDefault` - what do you think?

Reply:
Sure, `generateTemplateDefault` is quite a mouthful and is difficult to remember. `initSite` is definitely much better.